test/test_autodiff_2.cpp

```diff
@@ -536,7 +547,7 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(atan2_function, T, all_float_types) {
     auto y = y_sampler.next();
 
     auto autodiff_v = atan2(make_fvar<T, m>(x), make_fvar<T, m>(y));
-    auto anchor_v = atan2(x, y);
+    auto anchor_v = atan2_wrap(x, y);
```

Comment:
Just curious: what was the issue here?  Surely the `using std::atan2` should take care of things?